src/common/snippets/src/lowered/pass/optimize_domain.cpp

```diff
@@ -78,12 +78,13 @@ inline bool OptimizeDomain::can_increase_jit_work_amount(const VectorDims& maste
 }
 bool OptimizeDomain::run(snippets::lowered::LinearIR& linear_ir) {
     OV_ITT_SCOPED_TASK(ov::pass::itt::domains::SnippetsTransform, "Snippets::OptimizeDomain")
-    const auto& config = linear_ir.get_config();
-    if (linear_ir.empty())
+    // If m_tile_rank > 0, it means that backends manually set this value
```

Comment:
This is not supposed to work this way 🙂
We invested resources into PassConfig, so the plugin can disable any transformation.
This seems like a perfect use case for this feature.

Reply:
Fully agree. Removed